waterloop-site/src/theme/global.scss

```diff
@@ -200,13 +200,64 @@ h1 {
   text-align: justify;
   width: 50%;
   height: 100%;
-  padding-right: 3%;
   display: flex;
   flex-direction: column;
-  margin-bottom: 3em;
+}
```

Comment:
Rearranged CSS selectors to allow selectors in media query to overwrite these ones.